### PR TITLE
navigation added

### PR DESCRIPTION
**Pull request by Sharmaryan**
Screenshot (21)

added navigation , not responsive

### CODE REVIEW

File: styles/navigation/nav.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
+  --default: rgb(53, 52, 52);
+}
+.nav-container {
+  padding: 1rem;
```
Comment (Kavita539) [inline styles/navigation/nav.css:14]: suggestion: : You can use global variables for spacing to have consitency throughout the code.

File: styles/navigation/nav.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: white;
+  --secondary-color: rgb(15, 1, 1);
```
Comment (jsuryakt) [inline styles/navigation/nav.css:10]: Can declare variables for colors

**Review by Kavita539 — APPROVED** [review]

**Comment by Kavita539** [comment]
It looks good!

File: styles/navigation/nav.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/navigation/nav.css:14]: definitely, will do that

**Review by jsuryakt — APPROVED** [review]

**Review by chanchal-panpaliya — APPROVED** [review]
Code looks fine... Html is really well presented.

File: styles/navigation/nav.css (hunk 2 above)
Comment (Sharmaryan) [reply styles/navigation/nav.css:10]: will refactor for every component, thanks

**Comment by preethasri** [comment]
For desktop UI looks neat and good 👍

**Pull request merged**